app/backend/Extensions/KeyVaultConfigurationBuilderExtensions.cs

```diff
@@ -6,11 +6,13 @@ internal static class KeyVaultConfigurationBuilderExtensions
 {
     internal static IConfigurationBuilder ConfigureAzureKeyVault(this IConfigurationBuilder builder)
     {
-        var azureKeyVaultEndpoint = Environment.GetEnvironmentVariable("AZURE_KEY_VAULT_ENDPOINT") ?? throw new InvalidOperationException("Azure Key Vault endpoint is not set.");
-        ArgumentNullException.ThrowIfNullOrEmpty(azureKeyVaultEndpoint);
-
-        builder.AddAzureKeyVault(
-            new Uri(azureKeyVaultEndpoint), new DefaultAzureCredential());
+       // var azureKeyVaultEndpoint = Environment.GetEnvironmentVariable("AZURE_KEY_VAULT_ENDPOINT") ?? throw new InvalidOperationException("Azure Key Vault endpoint is not set.");
```

Comment:
it seems like we can take this out, disable this for future

Reply:
its because the values in keyvault are not sensitive and we have been hardcoding them

like the name of our container or name of ai search index yada yada